BraveShared/de.lproj/BraveShared.strings

```diff
@@ -844,9 +844,6 @@
 /* Add a Computer title */
 "SyncAddComputerTitle" = "Einen Computer hinzufügen";
 
-/* Add mobile device to sync with scan */
-"SyncAddDeviceScan" = "QR-Code der Kette scannen";
```

Comment:
Note: these two removed strings have updated recently and haven't been re-translated yet.
They will be re-added later after new translations are in